sqlglot/expressions.py

```diff
@@ -233,6 +234,25 @@ def transform(self, fun, *args, copy=True, **kwargs):
         return new_node
 
 
+class Term(Expression):
```

Comment:
why do we need another subclass, can we just put this in expression?

Reply:
We won't need this anymore with the direction we're going.